src/Catty/DataModel/Project/Project.m

```diff
@@ -377,7 +212,7 @@ - (NSString*)description
     [ret appendFormat:@"Screen Height: %@\n", self.header.screenHeight];
     [ret appendFormat:@"Screen Width: %@\n", self.header.screenWidth];
     [ret appendFormat:@"Screen Mode: %@\n", self.header.screenMode];
-    [ret appendFormat:@"Sprite List: %@\n", self.objectList];
+    [ret appendFormat:@"Sprite List: %@\n", self.scene.objects];
```

Comment:
Instead of the Sprite List, print the `Scene` and override the description in this class (to display the objects).